events/elections/2018/README.md

```diff
@@ -2,63 +2,91 @@
-The top vote getters, modulo the corporate diversity requirement of no more than 1/3 of the committee from a single company, will be elected to the respective positions.
+- [elegibility for candidacy]
+- [eligibility for voting]
 
-You will be ranking your choices of the candidates with an option for "no opinion". In the event of a tie, a coin will be flipped.
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| August 15    | Announcement of Election |
+| September 14 | All candidate bios / statements of intent due. Voting exemption forms due. |
+| September 19 | Election Begins |
+| October 3    | Election Closes |
+| October 4    | Announcement of Results |
+
+## Candidacy Process
+
+Candidates must submit a pull request with a biography in this directory with
+their platform and intent to run. This statement is **limited to 300 words**
+and must follow the format of `firstnamelastname.md`. Please refer to the
+[2017 candidate bios] for examples.
+
+## Voting Process
 
-The election will open for voting on September 19, 2018 at 04:00pm UTC and end two weeks after on October 3, 2018 at 01:00am UTC. You will receive an email to the address on file at the start of the election from Jorge Castro (jorge@heptio), Community Manager, please whitelist if necessary. Detailed voting instructions will be addressed in email and the CIVS polling page.
+Elections will be held using time-limited [Condorcet] ranking on [CIVS]
+using the [Schulze method]. The top vote getters will be elected to the open
+seats.
+
+Employer diversity is encouraged, and thus [maximal representation] will be
+enforced as spelled out in the [Steering Committee Election Charter].
+
+You will be ranking your choices of the candidates with an option for
+"no opinion". In the event of a tie, a coin will be flipped.
+
+The election will open for voting on September 19, 2018 at 04:00pm UTC and
+end two weeks after on October 3, 2018 at 01:00am UTC. You will receive an
+email to the address on file at the start of the election from Jorge Castro
+(jorge@heptio), Community Manager, please whitelist if necessary. Detailed
+voting instructions will be addressed in email and the CIVS polling page.
 
 ### Officers
 
-The Steering Committee has selected the following people as [election officers](https://github.com/kubernetes/community/tree/master/events/elections#election-officers):
+The Steering Committee has selected the following people as [election officers]:
 - Paris Pittman, @parispittman, Google
 - Ihor Dvoretskyi, @idvoretskyi, CNCF
 - Jorge Castro, @castrojo, Heptio
 
-### Schedule
-
-August 15 - Announcement of Election
-September 14 - All candidate bios / statement of intents due. Voting exemption forms due.
-September 19 - Election Begins
-October 3 - Election Closes
-October 4 - Announcement of Results
-
-### Eligibility  
-Members of Standing are defined by the union of:
```

Comment:
Members of Standing is kind of meaningless, since it's just used to decide "who can vote".  I broke this section up into Candidacy Process and Voting Process.  The eligibility requirements are spelled out in kubernetes/steering

Candidates don't technically have to meet the eligibility requirements that voters do.  We can change that if we think it's necessary.